python/ray/data/_internal/numpy_support.py

```diff
@@ -7,6 +8,8 @@
 from ray.air.util.tensor_extensions.utils import create_ragged_ndarray
 from ray.data._internal.util import _truncated_repr
 
+logger = logging.getLogger(__file__)
```

Comment:
nit: we typically use `logging.getLogger(__name__)` as convention in Ray Data. any particular choice for `__file__`?